scripts/helm/delete

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
there's no mention of the helm delete and install scripts in the README. The documentation should make it clear a build must be run prior to deploy

Reply:
The Readme did mention the old `deploy` script and showed build then deploy in its example for how to test Helm deployments locally. I've updated it to refer to the `install` and `delete` scripts which replaced `deploy`.

Reply:
Good work on the helm deploy scripts, very useful to be able to deploy to a cluster and kill of the queue and database to see the probes in action